charts/testkube/values-demo.yaml

```diff
@@ -154,7 +154,8 @@ testkube-api:
 
         if ( $request_method = POST ){
           set $methodallowed "true";
-        } 
+          set $pathallowed "true";
```

Comment:
just in case, we allow POST for any path in this case

Reply:
update config to allow only repositories